pkg/apis/k0s.k0sproject.io/v1beta1/kubeproxy.go

```diff
@@ -30,9 +31,11 @@ const (
 
 // KubeProxy defines the configuration for kube-proxy
 type KubeProxy struct {
-	Disabled           bool   `json:"disabled,omitempty"`
-	Mode               string `json:"mode,omitempty"`
-	MetricsBindAddress string `json:"metricsBindAddress,omitempty"`
+	Disabled           bool                            `json:"disabled,omitempty"`
+	Mode               string                          `json:"mode,omitempty"`
+	MetricsBindAddress string                          `json:"metricsBindAddress,omitempty"`
+	IPTables           map[string]runtime.RawExtension `json:"iptables,omitempty"`
```

Comment:
Why `runtime.RawExtension` type?

Reply:
This is the only working solution I could come up with. There is no way to use `interface{}` in the CRDs.